content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -285,6 +285,12 @@ and installing the sidecar injector webhook configuration on the remote cluster
     $ istioctl manifest generate -f external-istiod.yaml | kubectl apply --context="${CTX_EXTERNAL_CLUSTER}" -f -
     {{< /text >}}
 
+    At last, clean `istiod-default-validator` webhook:
```

Comment:
Can you better explain what this is doing. In this part of the doc we are setting up the control plane. Then it mentions we are `cleaning` a web hook by deleting it. I suspect there should be some words related to why the web hook should not be there and we should delete it followed by the kubectl delete.

Reply:
this doc guide user install two istiod(one in istio-system namesapce, another in external-istiod namespace) in external cluster via `istioctl install`
without https://github.com/istio/istio/pull/38387 `istiod-default-validator` webhook always point to `istiod.istio-system.svc:443` even you change `global.istioNamespace`.

Reply:
Explain in the document text, what is happening.